### DISCUSSION TITLE
cursor not restting/recalibrating/moving when picking the pen up

### DISCUSSION THREAD

**Discussion by thekoboldomancer** [ppy/osu#25037] [Q&A]
i made a post on the osu.ppy.sh forms and was told to come here
when i hover my pen above my tablet, the cursor moves. however, when picking up the pen a placing in another point on the tablet's surface, the cursor does not jump to the position of the pen. moving the pen at that point causes the cursor to moved in relation to the last point the cursor was located. this issue only occurs when tabbed into osu. it is not an issue of drivers or tablet software, as i have uninstalled both 3+ times. opentabletdriver is not detecting my tablet either. i was using a Huion H610PROV2 with huion driver version v.15.7.6.485 and opentabletdriver v0.6.3.0.

**Comment by peppy** [comment]
It sounds like things are not running in absolute mode.

If OTD isn't detecting your tablet, that's the likely cause.
